tutorials/deploy-multitenant-app-kyma/deploy-multitenant-app-kyma.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Deploy a Multitenant Application to a Provider subaccount, Kyma Runtime
+description: Build a Node.js application into OCI image and push it into Docker registry. Based on that, deploy the application into the Kyma runtime.
+auto_validation: true
+time: 25
+tags: [ tutorial>intermediate, software-product>sap-business-technology-platform]
+primary_tag: software-product>sap-btp\, kyma-runtime
+---
+
+## Prerequisites
+- You have a Kyma runtime environment on SAP Business Technology Platform (BTP). If not, please follow this tutorial: [Enable SAP BTP, Kyma Runtime](cp-kyma-getting-started).
+- You have installed [Docker](https://docs.docker.com/get-started/#download-and-install-docker).
+- You have [Docker Hub](https://hub.docker.com/) account.
+- You have installed [Kubernetes command-line tool](https://kubernetes.io/docs/tasks/tools/#kubectl).
+- You have finished the tutorial [Register a Multitenant Application to the SAP SaaS Provisioning Service](register-multitenant-app-saas-provisioning-service).
+
+
+
+## Details
+### You will learn
+- How to build Application to OCI Image
+- How to push OCI image to Docker Hub
+- How to deploy applications into Kyma runtime
+
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Determine SAP BTP Subaccount Subdomain)]
+
+Open your subaccount in the Cockpit. In the overview page, find the subdomain for your deployment.
+
+For example:
+
+![image-20211214133316133](image-20211214133316133.png)
```

Comment:
Please add borders to your images.

Reply:
What kind of border do you mean, may you please give a sample?